hack/update-fluentd-vendor-gems.sh

```diff
@@ -68,17 +68,10 @@ while read gemname gemver ; do
         grep github.com $vendordir/$gemname.gemspec
         exit 1
     fi
-    if type -p brew > /dev/null 2>&1 ; then
-        if [ $gemname = fluentd ] ; then
-            pkgname=fluentd
-        else
-            pkgname="rubygem-${gemname}"
-        fi
-        if foundname=$( brew search package --exact $pkgname ) && [ "$foundname" = "$pkgname" ] ; then
-            : # package exists in brew
-        else
-            pkgname=$gemname
-        fi
+    if [ $gemname = fluentd ] ; then
+        pkgname=fluentd
+    else
+        pkgname="rubygem-${gemname}"
```

Comment:
do we need this?  This I think is a holdover of gems being packaged in RPMs which would be unnecessary if we are vendoring and no longer relying on brew

Reply:
OMG, i thought this is a hindsight of someone using brew on a macOS machine.

Reply:
I fixed this script for local dev machines, where brew is missing.